crates/node/p2p/src/gossip/driver.rs

```diff
@@ -105,17 +105,17 @@ impl GossipDriver {
     /// Dials the given [`Multiaddr`].
     pub fn dial_multiaddr(&mut self, addr: Multiaddr) {
         if self.has_dialed(&addr) {
-            debug!(target: "p2p::gossip::driver", "Already connected to peer: {:?}", addr.clone());
+            trace!("Already connected to peer: {:?}", addr.clone());
```

Comment:
why remove the target filter? 
```suggestion
            trace!(peer=%addr
                "Already connected to peer"
            );
```

Reply:
It's already registered and printed by module path